dotnet/src/SemanticKernel.Skills/Skills.Web/WebSearchEngineSkill.cs

```diff
@@ -20,14 +40,35 @@ public WebSearchEngineSkill(IWebSearchEngineConnector connector)
 
     [SKFunction("Perform a web search.")]
     [SKFunctionInput(Description = "Text to search for")]
+    [SKFunctionContextParameter(Name = Parameters.Count, Description = "Number of results")]
```

Comment:
Let's add default values of count = 1 and offset = 0, to support the default case. This will help the Kernel and language models to understand what it means to not set these parameters.